src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+} //
```

Comment:
This interface should have a method:
```suggestion
public interface AreaCalculator {
    double getArea();
} 
```